core/src/main/java/io/servicecomb/core/Const.java

```diff
@@ -34,5 +34,5 @@ private Const() {
 
     public static final String SRC_MICROSERVICE = "x-cse-src-microservice";
 
-    public static final String DEST_MICROSERVICE = "x-cse-dest-microservice";
+    public static final String TARGET_MICROSERVICE = "x-cse-target-microservice";
```

Comment:
when target process have multiple microserives,  and consumer do not use cse sdk
then consumer must set x-cse-target-microservice in header